CraftTweaker2-API/src/main/java/crafttweaker/api/entity/IEntityFishHook.java

```diff
@@ -0,0 +1,31 @@
+
+package crafttweaker.api.entity;
+
+import crafttweaker.annotations.ZenRegister;
+import crafttweaker.api.player.IPlayer;
+import stanhebben.zenscript.annotations.*;
+
+@ZenClass("crafttweaker.entity.IEntityFishHook")
+@ZenRegister
+public interface IEntityFishHook extends IEntity {
```

Comment:
Perhaps add an AT to get the `currentState` of the hook? Also a setDead method.

Reply:
currentState is private and I don't think setDead is needed because it's implemented in IEntity

Reply:
> currentState is private

which is why i suggested an AT